use-cases.md

```diff
@@ -180,7 +180,8 @@ The client and server examples above propagated the Span/Trace over the wire, in
     span.set_trace_attribute('auth-token', '.....')
 
     # server side (one or more levels down from the client)
-    token = span.get_trace_attribute('auth-token')
+    h_ctx, h_attr = tracer.propagate_span_as_text(span)
+    token = h_attr.get_trace_attribute('auth-token')
```

Comment:
this does not remove Heisenberg effect, since the end app still can get the attribute, it just makes the API harder to use.